gno.land/pkg/gnoweb/components/view_help.go

```diff
@@ -14,10 +15,11 @@ type HelpData struct {
 	SelectedArgs map[string]string
 
 	RealmName string
-	Functions []vm.FunctionSignature
+	Functions []*doc.JSONFunc
 	ChainId   string
 	Remote    string
 	PkgPath   string
+	Doc       string `json:"package_doc"`
```

Comment:
```suggestion
	Doc       string
```

The `HelpData` struct isn't exported to JSON, so it doesn't need JSON field names.